docs/topics/components-stability.md

```diff
@@ -70,9 +70,9 @@ Kotlin/JS (Classic back-end)|Stable|1.3| |
 Kotlin/JVM (IR-based)|Stable|1.5| |
 Kotlin/JS (IR-based)|Beta|1.5| |
 Kotlin/Native Runtime|Beta|1.3| |
-Kotlin/Native new memory manager|Alpha|1.6.20| |
+Kotlin/Native new memory manager|Beta|1.7.20| |
 klib binaries|Alpha|1.4| |
-Multiplatform Projects|Alpha|1.3| |
+Multiplatform Projects|Beta|1.7.20| |
```

Comment:
I don't think Kotlin Multiplatform is in Beta, only Mobile is
Or am I missing something?

Reply:
I suppose that most of the Multiplatform status depends on the Mobile state.
Let me double-check that